BLSQ/iaso
# WC2-308: Fix exit types

For the data genereated by the ETL script for tableau dashboard, there were some missing exit types for a beneficiary in the journey database table.

Related JIRA tickets : [WC2-308](https://bluesquare.atlassian.net/browse/WC2-308)

## Self proofreading checklist


## Changes
 
For the discharged beneficiaries, you should see the exit type in their journey. 

## How to test

To test it, user should have admin access. From the WFP admin Journey page, try to create a new journey and check in the **Exit type** drop down list if there are all possible elements in the form.

## Print screen / video


https://github.com/BLSQ/iaso/assets/6383219/3df723f2-660e-47c3-b9b4-a3396e6e1b5c




[WC2-308]: https://bluesquare.atlassian.net/browse/WC2-308?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
